packages/components/src/components/comparison-table/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ComponentProps, FC } from 'react';
+import { cn } from '@theguild/components';
+
+const Table: FC<ComponentProps<'table'>> = ({ className, ...props }) => {
+  return (
+    <table
+      className={cn(
+        'x:block x:overflow-x-auto nextra-scrollbar overflow-x-auto rounded-2xl border border-green-200',
+        className,
+      )}
+      {...props}
+    />
+  );
+};
+
+const TableRow: FC<ComponentProps<'tr'> & { highlight?: boolean }> = ({
+  highlight,
+  className,
+  ...props
+}) => {
+  return <tr className={cn(highlight && 'bg-green-100', className)} {...props} />;
+};
+
+const cellStyle = cn(
+  'border border-green-200 p-4 first:border-l-0 last:border-r-0',
+  '[tbody_&]:border-b-0 [thead_&]:border-t-0',
+  'first:sticky',
+  'first:left-0',
+  'max-sm:first:drop-shadow-2xl',
+  'first:bg-[rgb(var(--nextra-bg))]',
+);
+
+const TableHeader: FC<ComponentProps<'th'>> = ({ className, ...props }) => {
+  return <th className={cn(cellStyle, 'font-medium', className)} {...props} />;
+};
+
+const TableCell: FC<ComponentProps<'td'>> = ({ className, ...props }) => {
+  return <td className={cn(cellStyle, className)} {...props} />;
+};
+
+export const ComparisonTable = Object.assign(Table, {
```

Comment:
I think we could

```tsx
import { Table } from "@theguild/components"
```

```suggestion
export const Table = Object.assign(_Table, {
```

It doesn't really have to be a comparison table. It is one in Mesh, maybe in Yoga too, but I think the styles work for every table.

Reply:
in components `<Table>` component already reexported from Nextra https://github.com/the-guild-org/docs/blob/e60c4c6e6a09a442203ec20233fd33be2a58f955/packages/components/src/index.ts#L10, should we override the default Table for mdx pages too?